docs/sources/operators-guide/secure/securing-communications-with-tls.md

```diff
@@ -126,3 +129,35 @@ The following example shows how to configure the gRPC client flags in the querie
     # Path to the TLS CA for the gRPC Client
     -querier.frontend-client.tls-ca-path=/path/to/root.crt
 ```
+
+##### Available cipher suites
+
+The `*.tls-cipher-suites` configuration options allow to override the default list of accepted TLS cipher suites.
+The configuration option value should be a comma-separated list of accepted suites.
+The TLS cipher suites supported by Mimir are:
+
+- `TLS_RSA_WITH_AES_128_CBC_SHA`
+- `TLS_RSA_WITH_AES_256_CBC_SHA`
+- `TLS_RSA_WITH_AES_128_GCM_SHA256`
+- `TLS_RSA_WITH_AES_256_GCM_SHA384`
+- `TLS_AES_128_GCM_SHA256`
+- `TLS_AES_256_GCM_SHA384`
+- `TLS_CHACHA20_POLY1305_SHA256`
+- `TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA`
+- `TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`
+- `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA`
+- `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA`
+- `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`
+- `TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384`
+- `TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`
+- `TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384`
+- `TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256`
+- `TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256`
+- `TLS_RSA_WITH_RC4_128_SHA` (insecure)
+- `TLS_RSA_WITH_3DES_EDE_CBC_SHA` (insecure)
+- `TLS_RSA_WITH_AES_128_CBC_SHA256` (insecure)
+- `TLS_ECDHE_ECDSA_WITH_RC4_128_SHA` (insecure)
+- `TLS_ECDHE_RSA_WITH_RC4_128_SHA` (insecure)
+- `TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA` (insecure)
+- `TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256` (insecure)
+- `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256` (insecure)
```

Comment:
I don't think we should include this list in the generic documentation. It can change between Go versions, and we may not update it properly.

Reply:
It's a valid concern. What's about https://github.com/grafana/mimir/pull/3070/commits/2a511447cc5f8ab25e12263e029b4bcea911c9a9 ?

Reply:
Looks good, thanks!